.gitignore

```diff
@@ -1,6 +1,9 @@
 ### Linux ###
 *~
 
+# demo output directory
+YOLOX_outputs/
```

Comment:
Could you plz add datasets directory also? Thx

Reply:
Sure, makes sense!